runtime/syntax/python.vim

```diff
@@ -80,6 +81,7 @@ syn keyword pythonRepeat	for while
 syn keyword pythonOperator	and in is not or
 syn keyword pythonException	except finally raise try
 syn keyword pythonInclude	from import
+syn keyword pythonAsync		async await
```

Comment:
You have tabs here used for alignment, but they put keywords too far. Note that in Vim you need to set `ts=8` to get correct view. GH also uses ts=8 here.